web/src/components/alerting/AlertsPage.tsx

```diff
@@ -155,21 +164,38 @@ const AlertsPage_: React.FC<AlertsPageProps> = () => {
       },
       {
         id: 'actions',
-        props: { className: tableAlertClasses[4] },
+        props: { className: tableAlertClasses[5] },
         title: '',
       },
-    ],
-    [t],
-  );
+    ];
+
+    if (perspective === 'acm') {
+      cols.splice(-1, 0, {
+        id: 'cluster',
+        props: { className: tableAlertClasses[4] },
+        sort: 'labels.cluster',
+        title: t('Cluster'),
```

Comment:
maybe not for this PR, but this should probably be a link to the alert in the spoke